lib/shared/constants.js

```diff
@@ -30,8 +31,7 @@ const SUBTITLE_SCHEMA = Joi.object().keys({
 });
 
 const PARAM_SCHEMA = Joi.object().keys({
-  subtitleText: Joi.string().required(),
-  inputExtension: Joi.string().required(),
+  subtitleText: Joi.string().regex(ALL_VALID_EXT_REGEX).required().error(() => 'Input file type is not supported.'),
```

Comment:
Actually it looks like this will be the error thrown if there is an invalid file type.  This works.